environment.yml

```diff
@@ -21,14 +21,14 @@ dependencies:
 - kiwipy[rmq]~=0.7.5
 - importlib-metadata~=4.3
 - numpy~=1.19
-- pamqp~=2.3
```

Comment:
this was not necessary before? or are we removing code that depended on it?

Reply:
We needed it to provide support for RabbitMQ 3.5 but I removed this (see the second commit in this PR). This was its only use so now we no longer need it and can remove it.